src/app/components/quick-pick/quick-pick.component.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../assets/sass/variables";
+@import "../../../assets/sass/_theme-variables";
```

Comment:
There is no need to import _theme-variables, this file is already imported at the end of variables.scss